.github/workflows/releaser.yml

```diff
@@ -8,31 +8,23 @@ on:
 
 jobs:
 
-  # awx compatible binary with correct version of gcc and clang
-  binary_centos7_amd64:
+  binary_linux_amd64:
     runs-on: ubuntu-latest
-    container:
-      image: amd64/centos:centos7
     steps:
          - uses: actions/checkout@v2
          - name: install cargo deps and build avail
+           shell: bash
            run: |
-            yum install centos-release-scl -y
-            yum install llvm-toolset-7 -y
-            yum install gcc-c++ -y
-            scl enable llvm-toolset-7 bash
-            export PATH="/opt/rh/llvm-toolset-7/root/usr/bin:$PATH"
-            export LIBCLANG_PATH="/opt/rh/llvm-toolset-7/root/usr/lib64/"
             curl https://sh.rustup.rs -sSf | sh -s -- -y
             source "$HOME/.cargo/env"
             cargo build --release -p data-avail
-            mv target/release/data-avail target/release/data-avail-centos7-amd64
+            mv target/release/data-avail target/release/data-avail-linux-amd64
          - uses: actions/upload-artifact@v2
            with:
-             name: data-avail-centos7-amd64-binary
-             path: target/release/data-avail-centos7-amd64
-
-  binary_linux_amd64:
+             name: data-avail-linux-amd64-binary
```

Comment:
keep uncompressed amd64 version for now to prevent any downstream breakage, we will delete once ansible playbooks updated to unarchive tar version